src/RecoveryServices/RecoveryServices/Vault/GetAzureRmRecoveryServicesBackupProperty.cs

```diff
@@ -49,10 +50,10 @@ public override void ExecuteCmdlet()
         {
             try
             {
-                BackupStorageConfig getStorageResponse = RecoveryServicesClient.GetVaultStorageType(
+                BackupResourceConfigResource getStorageResponse = RecoveryServicesClient.GetVaultStorageType(
```

Comment:
Please try to use BackupResourceConfig as the replacement for BackupStorageConfig being used earlier

Reply:
We are consuming the BackupResourceConfig of BackupResourceConfigResource(properties field) returned from the method GetVaultStorageType

Reply:
Okay, then it's fine